assets/stylesheets/components/_media.scss

```diff
@@ -0,0 +1,33 @@
+.media-card__container {
+  @extend %clearfix;
+
+  & + * {
+    margin-top: 0;
+  }
+}
+
+.media-card {
```

Comment:
Why is the file called `components/media` and the CSS selector is called `.media-card`?